### DISCUSSION TITLE
What initial user to log in with?

### DISCUSSION THREAD

**Discussion by olibuijr** [TomBursch/kitchenowl#176] [Q&A]
I created the instance from docker-compose.yml

What username and password are default?

Kind regards

**Comment by TomBursch** [comment]
There was a bug since yesterday that broke some user stuff. If you upgrade to the newest version you should be greeted with an onboarding screen telling you to create a user.

**Reply by g5pw** [reply]
I'm still hitting this on a new install, I get redirected to /signin

EDIT: uh, scratch that, sorry. It apparently already had  the data volume from a previous install I forgot about :D
